<?xml version="1.0" encoding="UTF-8"?>
<launch>

   <node name="coral_test"	pkg="tf_ros_detection"      type="coral_test"  	respawn="true" 	output="screen"><!-- launch-prefix="xterm -e gdb !args"-->
                <remap from="cam_in" 			to="/stereo_cam_node/image_raw"/>
		<param name="labels_dir"	type="string" 	value="/mscoco_label_map2.pbtxt" /> 
		<param name="models_dir"	type="string" 	value="/home/ubuntu/catkin_p3/src/test_p3/src/models/mobilenet/posenet_mobilenet_v1_075_481_641_quant_decoder_edgetpu.tflite" />
		<!-- /home/ubuntu/Downloads/coco_mobile/detect.tflite , /home/ubuntu/Downloads/facenet_keras.tflite  , /home/ubuntu/Downloads/62faces_float.tflite -->
 		<param name="threshold_Score" value="1.5" />
 		<param name="threshold_IOU" value="0.4" />
     </node>
  
   <node name="pose_decision"	pkg="tf_ros_detection"      type="pose_decision"  	respawn="true" 	output="screen"><!-- launch-prefix="xterm -e gdb !args"-->
                <remap from="cam_in" 			to="/stereo_cam_node/image_raw"/>
		<param name="labels_dir"	type="string" 	value="/mscoco_label_map2.pbtxt" /> 
		<param name="models_dir"	type="string" 	value="/home/ubuntu/Downloads/coco_mobile/detect.tflite" />
		<!-- /home/ubuntu/Downloads/coco_mobile/detect.tflite , /home/ubuntu/Downloads/facenet_keras.tflite  , /home/ubuntu/Downloads/62faces_float.tflite -->
 		<param name="threshold_Score" value="1.5" />
 		<param name="threshold_IOU" value="0.4" />
 		<param name="delay" value="1" />
     </node>
     
</launch>
